audit-dotnet-core/.gitlab-ci.yml

```
@@ -28,3 +28,6 @@ jfrog-nuget-audit:
 
     # Audit .NET project
     - jf audit
+
+    # Cleanup
+    - !reference [.cleanup_jfrog, script]
```

Should be in an after_script tag:
```suggestion
  after_script:
      # Cleanup
    - !reference [.cleanup_jfrog, script]
```
Let's do it in all templates